packages/libp2p/src/connection-manager/utils.ts

```diff
@@ -69,7 +73,26 @@ export function combineSignals (...signals: Array<AbortSignal | undefined>): Cle
   try {
     // fails on node < 15.4
     setMaxListeners?.(Infinity, signal)
-  } catch {}
+  } catch { }
 
   return signal
 }
+
+export const validateConnectionManagerConfig = (opts: ConnectionManagerInit): ObjectSchema<Record<string, unknown>> => {
+  return object({
+    maxConnections: number().min(opts?.minConnections ?? MIN_CONNECTIONS, `maxConnections must be greater than the min connections limit: ${opts?.minConnections}`).integer().default(MAX_CONNECTIONS),
```

Comment:
```suggestion
    maxConnections: number().min(opts?.minConnections ?? MIN_CONNECTIONS, `maxConnections must be greater than minConnections: ${opts?.minConnections}`).integer().default(MAX_CONNECTIONS),
```

I think these error messages should use the names of the related options rather than a description of them, eg: minConnections, rather than 'the min connections limit'.
Applies to all of these error messages